### DISCUSSION TITLE
Feature Request: Supercharged Wikilinks

### DISCUSSION THREAD

**Discussion by rktic** [svsool/vscode-memo#571] [Ideas]
Hej Svyat,

just stumbled upon Memo - thank you for making this! I'm seriously considering replacing Obsidian with VS Code. Because of Obsidians stubborn nature of handling links. Background:

Wikilinks are great for writing and reading. At the same time, when it comes to publishing content, most SSGs expect standard markdown links. In order to differentiate between internal and external links, some require customized standard markdown links. To give you an example: zola expects @/link-destination for handling them correctly. 

Lots of not-so-tech-savvy folks like me (who focus on writing, not coding) are limited by how their note-taking tool handles links. Here's how this could be solved:

Write and display links as Wikilinks, but actually write standard markdown links into the file. With an option to customize them in the settings. To eg add SSG specific semantic to them. Would that be do-able?

It'd be a huge life-saver for me. And, after doing a year of researching SSGs now: for many others as well.

Cheers to Helsinki
Ronny

**Comment by svsool** [comment]
Hi Ronny,

It sounds like you might want to try an exporter like [obsidian-export](https://github.com/zoni/obsidian-export).
